Content.Shared/Teleportation/Systems/SharedPortalSystem.cs

```diff
@@ -206,6 +210,10 @@ private void OnEndCollide(EntityUid uid, PortalComponent component, ref EndColli
 
         _transform.SetCoordinates(subject, target);
 
+        TryComp<AnomalyComponent>(subject, out var anomaly);
+        if(anomaly != null)
+           _anomalySystem.DoAnomalySupercriticalEvent(subject);
+
```

Comment:
```suggestion
        if (HasComp<AnomalyComponent>(subject))
           _anomalySystem.DoAnomalySupercriticalEvent(subject);

```
Не хочу душить, но так получше выглядит, все равно `anomaly` дальше не используется.